externals/kyuubi-flink-sql-engine/src/main/scala/org/apache/kyuubi/engine/flink/schema/RowSet.scala

```diff
@@ -377,4 +398,23 @@ object RowSet {
         other.toString
     }
   }
+
+  /** should stay in sync with [[org.apache.hadoop.hive.common.type.TimestampTZUtil]] */
+  var TIMESTAMP_LZT_FORMATTER: DateTimeFormatter = {
+    val builder = new DateTimeFormatterBuilder()
+    // Date part
+    builder.append(DateTimeFormatter.ofPattern("uuuu-MM-dd"))
+    // Time part
+    builder.optionalStart().appendLiteral(" ").append(
+      DateTimeFormatter.ofPattern("HH:mm:ss")).optionalStart().appendFraction(
+      ChronoField.NANO_OF_SECOND,
+      1,
+      9,
+      true).optionalEnd().optionalEnd()
+    // Zone part
+    builder.optionalStart().appendLiteral(" ").optionalEnd()
+    builder.optionalStart().appendZoneOrOffsetId().optionalEnd()
```

Comment:
if it's not intended, why add this?

Reply:
Because parsing at JDBC requires the zone part, otherwise we would ran into this exception: https://github.com/apache/kyuubi/blob/06dd7d3754ee210f7744e46030965a6f5df1953b/kyuubi-hive-jdbc/src/main/java/org/apache/kyuubi/jdbc/hive/common/TimestampTZUtil.java#L84